### PR TITLE
Adding new software to conda-forge:  BELT (BEam Longitudinal Tracking)

### PR DESCRIPTION
**Pull request by MichaelEhrlichman**
Checklist
- [x ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x ] Source is from official source.
- [x ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x ] If static libraries are linked in, the license of the static library is packaged.
- [x ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x ] Build number is 0.
- [x ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by MichaelEhrlichman** [comment]
@conda-forge/staged-recipes, small, scientific code written in fortran90 ready for review!

**Comment by ChristopherMayes** [comment]
@conda-forge/staged-recipes this is ready for review

File: recipes/belt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{% set name = "BELT" %}
+{% set version = "BELTv1.0" %}
```
Comment (xhochy) [inline recipes/belt/meta.yaml:2]: Please use a numeric version
```suggestion
{% set version = "1.0" %}
```

File: recipes/belt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+              forward or backward tracking through an electron linear accelerator. 
+              It includes longitudinal space-charge effect, structure and 
+              resistive wall wakefields, longitudinal coherent synchrotron 
+              radiation effect.
+              Copyright Notice
+              BEam Longitudinal Tracking (BELT) Copyright (c) 2024, 
+              The Regents of the University of California, 
+              through Lawrence Berkeley National Laboratory (subject to receipt of 
+              any required approvals from the U.S. Dept. of Ener gy). All rights reserved.
+  license: BSD-4-Clause-UC
```
Comment (xhochy) [inline recipes/belt/meta.yaml:50]: The license actually looks like this one
```suggestion
  license: BSD-3-Clause-LBNL
```

File: recipes/belt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
+  script: 
+    - ln -sf $BUILD_PREFIX/bin/$CONDA_TOOLCHAIN_HOST-gfortran $BUILD_PREFIX/bin/gfortran
+    - cd src
+    - make -f Makefile
+    - mkdir -p $PREFIX/bin
+    - cp xeblt $PREFIX/bin
+    - cp -r ../examples/example1 $PREFIX/test
```
Comment (xhochy) [inline recipes/belt/meta.yaml:22]: Please don't put example data into the package.

File: recipes/belt/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  sha256: cbfe5081595ba91bb88e4cb9deec001f60c73571a8cc10788ac60d789089f843
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
+  script: 
+    - ln -sf $BUILD_PREFIX/bin/$CONDA_TOOLCHAIN_HOST-gfortran $BUILD_PREFIX/bin/gfortran
+    - cd src
+    - make -f Makefile
```
Comment (xhochy) [inline recipes/belt/meta.yaml:19]: Looks like this also might work
```suggestion
    - cd src
    - make FORTRAN=$FORTRAN
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/belt/meta.yaml (hunk 3 above)
Comment (MichaelEhrlichman) [reply recipes/belt/meta.yaml]: The test has been removed.

File: recipes/belt/meta.yaml (hunk 4 above)
Comment (MichaelEhrlichman) [reply recipes/belt/meta.yaml]: `make FORTRAN=$FORTRAN` did not work, but `make FORTRAN=$GFORTRAN` did work.

Comment (MichaelEhrlichman) [reply recipes/belt/meta.yaml:19]: I can find no solution other than the soft link that allows the code to compile on the conda-forge runners.

**Comment by MichaelEhrlichman** [comment]
@xhochy I made the changes you recommended, except for one regarting FORTRAN=$FORTRAN.   I have made comments in reply.

**Comment by MichaelEhrlichman** [comment]
@conda-forge/staged-recipes this is ready for review

**Review by MichaelEhrlichman — COMMENTED** [review]
Most changes implemented verbatim.  FORTRAN=$FORTRAN was problematic.  See comments in conversation.

**Comment by ChristopherMayes** [comment]
@conda-forge/staged-recipes this is ready for review again please

**Comment by ChristopherMayes** [comment]
@xhochy this should be ready for review again.

**Comment by MichaelEhrlichman** [comment]
@conda-forge/staged-recipes this is ready for review again please

File: recipes/belt/meta.yaml (hunk 4 above)
Comment (chrisburr) [reply recipes/belt/meta.yaml:19]: [Looks like you need](https://github.com/qianglbl/BELT/blob/main/src/Makefile#L28):

```
make CC=$FORTRAN LINK=$FORTRAN
```

Comment (MichaelEhrlichman) [reply recipes/belt/meta.yaml]: @chrisburr That works!  I have committed an update to the `make` command as suggested.

**Pull request merged**